### PR TITLE
[Workflows] add early termination form preview image

### LINKED ISSUE #2171: [Workflows] Add preview form image for ND early term

### PR DESCRIPTION
**Pull request by jovergaag**
## Description of the change

Adds the Early Termination form preview image to the client profile.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #2171 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally
- [ ] E2E have been run for Lantern changes ([Steps in the Readme](https://github.com/Recidiviz/pulse-dashboard#running-e2e-tests))

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/core/WorkflowsClientProfile/OpportunityPreview.tsx [hunk 1]
```diff
@@ -31,9 +32,9 @@ import { CriteriaList } from "./CriteriaList";
 import { OpportunityDenial } from "./OpportunityDenial";
 import { ClientWithOpportunityProps } from "./types";
 
-// TODO Remove the Partial here when we have an earlyTermination preview image
 const FORM_PREVIEW: Partial<Record<OpportunityType, string>> = {
```
Comment (macfarlandian) [inline src/core/WorkflowsClientProfile/OpportunityPreview.tsx:35]: remove the partial?

Comment (jovergaag) [reply src/core/WorkflowsClientProfile/OpportunityPreview.tsx]: wow, yes. thank you.

**Review by macfarlandian — APPROVED** [review]

File: src/core/WorkflowsClientProfile/OpportunityPreview.tsx [hunk 2]
```diff
@@ -23,6 +23,7 @@ import { Link } from "react-router-dom";
 import styled from "styled-components/macro";
 
 import compliantReportingFormPreviewSrc from "../../assets/static/images/compliantReportingFormPreview.png";
+import cearlyTerminationFormPreviewSrc from "../../assets/static/images/earlyTerminationFormPreview.png";
```
Comment (daschi) [inline src/core/WorkflowsClientProfile/OpportunityPreview.tsx:26]: Should this be `earlyTerminationFormPreviewSrc`?

**Review by daschi — COMMENTED** [review]
Is the preview a screenshot? It has a blue squiggly and a line thru the letter `S` in the first word...not a big deal and shouldn't block on merging just wanted to call it out

**Comment by jovergaag** [comment]
> Is the preview a screenshot? It has a blue squiggly and a line thru the letter S in the first word...not a big deal and shouldn't block on merging just wanted to call it out 

Fixed, good catch @daschi

**Pull request merged**